README.rst

```diff
@@ -58,6 +58,8 @@ To learn more about diffpy.pdffit2 library, see the examples directory
 included in this distribution or the API documentation at
 http://www.diffpy.org/doc/pdffit2.
 
+For more information about the diffpy.pdffit2 library, please consult our `online documentation <https://diffpy.github.io/diffpy.pdffit2>`_.
+
```

Comment:
Please look for a rogue slash in the curation below.

Also, please replace the installation instructions with the cookie cutter version. If there are special instructions for installing it from sources we can add them, but use the cookie cutter as a starting point with minimal updates and delete the old instructions.

Reply:
Perhaps we can make an issue for now to improve the installation instructions in reaadme once we have a PyPI/conda version available?

Reply:
No, do it now. We won't release until we have a puppy/cons version and then the instructions will work

Reply:
Instructions added.

Reply:
Done - rendered reamde: https://github.com/bobleesj/diffpy.pdffit2/tree/recookiecut